tests/tests/tier0/property-get/test_property_get.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import os
+import pytest
+from typing import Dict
+
+from bluechi_test.test import BluechiTest
+from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
+from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
+
+
+def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
+    result, output = ctrl.run_python(os.path.join("python", "get_property.py"))
+    if result != 0:
+        raise Exception(output)
+
+
+@pytest.mark.timeout(10)
```

Comment:
I would expect a larger timeout if possible, like 40 as a "new default".

Reply:
@dougsland 
As I've written here: https://github.com/containers/bluechi/pull/516#issuecomment-1702244638
I'll create a PR later and align all timeouts and introduce a scoped timeout for smaller code parts.

Reply:
Set it to 40 for now.